test/functional/comment_controller_test.rb

```diff
@@ -42,6 +42,29 @@ def teardown
     assert_template partial: 'questions/_comment'
   end
 
+  test 'should create wiki comments' do
+    UserSession.create(rusers(:bob))
+    assert_difference 'Comment.count' do
+      xhr :post, :create,
+          id: node(:wiki_page).nid,
+          body: 'Wiki comment'
+    end
+    assert_response :success
+    assert_not_nil :comment
+    #assert_template partial: 'wiki/_comment' 
+    #should uncomment above once comment displaying partial is implemented for wikis.
```

Comment:
But do we actually need regular comments in wiki pages? If this is a step to move to inline comments we can perhaps add a note for the same in code comments.